### DISCUSSION TITLE
Adding private notes

### DISCUSSION THREAD

**Discussion by joaquin102** [etsy/open-api#1064] [Q&A]
Hi everyone!
I read the entire documentation but i couldn't find a way to add private notes using the api.

is this even possible? if not, is there any plans to add it? This is extremely useful when multiple people are working at the same time.

Thank you!

**Comment by Gareth064** [comment]
Hey. It's not possible via API.

**Reply by Gareth064** [reply]
Doesn't help you now, but gaps like this, I am implementing into an app I have built (not public yet) which is basically like Etsy order management screen in their app but you can have multiple users (logged in) using it. You can assign order to users, add private notes etc. More advanced filtering etc.

Just not ready for use by the wider public yet. But getting there 🙃

**Reply by joaquin102** [reply]
Hey. It's unfortunate that it's not possible. So we have no other options to javascript/scrapping the way into it which is definitely no ideal.
That app sounds amazing! I would love to see it 👍

Thanks!

**Reply by Gareth064** [reply]
If your using the API then aren't you building an app as well? You could look to add the same features to yours, as I did 😉

Once my app is live I will give it a little plug here.

**Comment by duylddev** [comment]
it's 2025, could we fetch private notes on orders now?
